modules/logging-enabling-loki-alerts.adoc

```diff
@@ -74,23 +74,23 @@ The `AlertingRule` CR contains a set of specifications and webhook validation de
   kind: AlertingRule
   metadata:
     name: app-user-workload
-    namespace: app-ns <1>
-    labels: <2>
-      openshift.io/<label_name>: "true"
+    namespace: app-ns #<1>
+    labels: #<2>
+      openshift.io/cluster-monitoring: "true"
   spec:
-    tenantID: "application"
+    tenantID: application
     groups:
       - name: AppUserWorkloadHighError
         rules:
           - alert:
-            expr: | <3>
-            sum(rate({kubernetes_namespace_name="app-ns", kubernetes_pod_name=~"podName.*"} |= "error" [1m])) by (job)
+            expr: | #<3>
+              sum(rate({kubernetes_namespace_name="app-ns", kubernetes_pod_name=~"podName.*"} |= "error" [1m])) by (job)
             for: 10s
             labels:
-              severity: critical <4>
+              severity: critical #<4>
             annotations:
-              summary:  <5>
-              description:  <6>
+              summary:  #<5>
+              description:  #<6>
```

Comment:
Should we provide a value in the example?

```suggestion
              summary: This is an example summary #<5>
              description: This is an example description #<6>
```

Reply:
done